run_everything.R

```diff
@@ -0,0 +1,55 @@
+
+# ============================================================================ #
```

Comment:
we don't want a script in the root of the directory and I don't think we want it to have as a name run_everything. Ideally it would have a more informative name.